src/main/java/frc/robot/DriverJoystick.java

```diff
@@ -156,13 +159,9 @@ private DriverJoystick() {
      * @return sideways value
      */
     private double getSidewaysValue() {
-        double x = this.getLeftX() * maxPower;
-
-        double sideways = sidewaysFilter.calculate(x);
-        sideways = limitRate(x, prevSideways, sidewaysLimiter);
-        prevSideways = sideways;
+        prevSideways = limitRate(sidewaysFilter.calculate(this.getLeftX() * maxPower), prevSideways, sidewaysLimiter);
 
-        return sideways;
+        return prevSideways;
```

Comment:
```suggestion
        double x = this.getLeftX() * maxPower;

        double sideways = sidewaysFilter.calculate(x);
        sideways = limitRate(sideways, prevSideways, sidewaysLimiter);
        prevSideways = sideways;

        return sideways;
```
this is all you need to do to fix this. Chaining things together like you did makes it less readable